pass/helpers.go

```diff
@@ -17,11 +18,10 @@ func (c secretContainer) Bytes() []byte {
 }
 
 // generic function to be used in resource and data source
-func populateResourceData(d *schema.ResourceData, provider *passProvider, path string, readData bool) error {
-	st := provider.store
+func populateResourceData(d *schema.ResourceData, pp *passProvider, path string, readData bool) error {
```

Comment:
it's `pp` in every place except here, so I'm aligning it a bit.

Reply:
👍